components/markets/market-filter/MarketFiltersSort.tsx

```diff
@@ -0,0 +1,118 @@
+import { marketsOrderByOptions } from "lib/constants/market-filter";
+import { MarketOrderByOption, MarketsOrderBy } from "lib/types/market-filter";
+import { observer } from "mobx-react";
+import React from "react";
+import ReactSelect, { GroupBase, StylesConfig } from "react-select";
+import { useMarketFiltersContext } from "./MarketFiltersContainer";
+import { SelectComponents } from "react-select/dist/declarations/src/components";
+
+type MarketFilterSortProps = {
+  className?: string;
+  selectStyles?: StylesConfig<MarketOrderByOption, false>;
+  components?: Partial<
+    SelectComponents<MarketOrderByOption, false, GroupBase<MarketOrderByOption>>
+  >;
+};
+
+const sortBySelectStyles = {
+  control: (provided) => {
+    return {
+      ...provided,
+      width: "180px",
+      height: "32px",
+      minHeight: "32px",
+      fontSize: "14px",
+    };
+  },
+  dropdownIndicator: (provided) => {
+    return {
+      ...provided,
+      padding: "0px",
+      paddingRight: "10px",
+    };
+  },
+  singleValue: (provided) => {
+    return {
+      ...provided,
+    };
+  },
+  valueContainer: (provided) => {
+    return {
+      ...provided,
+      paddingLeft: "10px",
+    };
+  },
+  input: (provided) => {
+    return {
+      ...provided,
+    };
+  },
+  menu: (provided) => {
+    return {
+      ...provided,
+      backgroundColor: "white",
+      color: "black",
+      zIndex: 100,
+    };
+  },
+};
+
+const IndicatorSeparator = () => {
```

Comment:
Does this do anything?

Reply:
Yes, it removes vertical line between select value and indicator arrow that react select is showing by default. It is added in `components` prop of ReactSelect